### DISCUSSION TITLE
How should I bundle an AppImage for Linux?

### DISCUSSION THREAD

**Discussion by 1over137** [marcelotduarte/cx_Freeze#1770] [General]
I would like to provide an AppImage for my project (https://github.com/FreeLanguageTools/vocabsieve). I already currently have a cx_Freeze build job for Windows and Mac. However, I would like to distribute it as an AppImage to be able to use desktop integrations, so people don't have to risk breaking their python installations with `pip`, and it would take a while to package it for every distro.

**Comment by marcelotduarte** [comment]
Support for AppImage: bdist_appimage available #2054 
For other options, #1877
